go/api/base_client.go

```diff
@@ -213,3 +227,18 @@ func (client *baseClient) GetDel(key string) (string, error) {
 
 	return handleStringOrNullResponseWithFree(result), nil
 }
+
+func (client *baseClient) Ping(message ...string) (string, error) {
```

Comment:
```suggestion
func (client *baseClient) Ping(message string) (string, error) {
```
Only one argument is accepted by `ping`

Reply:
Taken note and updated this.